client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -195,9 +200,19 @@ function LineItemWrapper( {
 		// was in the cart when checkout finished loading (not necessarily the
 		// current variant). For WordPress.com only, not Jetpack or Akismet. See
 		// https://github.com/Automattic/wp-calypso/issues/69633
+
+		// Temporarily remove jetpack bi-yearly products so users can't purchase them for now.
```

Comment:
Just out of curiosity, since the available variants are provided by the shopping-cart endpoint, could these be removed there instead?